### PR TITLE
Adding github pages doc using mkdocs

### PR DESCRIPTION
**Pull request by edgarsilva948**
### What does this PR do?

implements the github pages documentation using mkdocs as suggested here #11  and also documents the destroy process to fix this #167 .

### Motivation

#11 

### More

- [ ] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/examples) to support my PR
- [ ] Yes, I have updated the [docs](https://github.com/aws-samples/crossplane-aws-blueprints/tree/main/doc) for this feature
- [ ] Yes, I have linked to an issue or feature request (applicable to PRs that solves a bug or a feature request)

**Note**:
 - Not all the PRs require examples and docs
 - We prefer small, well tested pull requests. Please ensure your pull requests are self-contained, and commits are squashed

### For Moderators
- [ ] E2E Test successfully complete before merge?

### Additional Notes
N/A

### CODE REVIEW

File: bootstrap/terraform/variables.tf [hunk 1]
```diff
@@ -16,7 +16,7 @@ variable "name" {
 variable "cluster_version" {
   type        = string
   description = "Kubernetes Version"
-  default     = "1.30"
+  default     = "1.29"
```
Comment (candonov) [inline bootstrap/terraform/variables.tf:19]: downgrading?

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Deploy
+
+### eksctl
+
+1. TBD
+
+### terraform
+
+1. For consuming Crossplane Blueprints, please see the [Consumption]() section.
```
Comment (candonov) [inline docs/getting-started.md:21]: is there a purpose to having this refer to an empty link?

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+1. TBD
+
+### terraform
+
+1. For consuming Crossplane Blueprints, please see the [Consumption]() section. 
+
+2. To bootstrap using terraform, the typical steps of execution are as follows:
+
+    ```sh
```
Comment (candonov) [inline docs/getting-started.md:25]: should we specify the folder with tf code?

File: docs/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    terraform apply -target="module.eks_blueprints_addons" -auto-approve
+    terraform apply -target="module.crossplane" -auto-approve
+    terraform apply -auto-approve
+    ```
+
+3. Once all of the resources have successfully been provisioned, the following command can be used to update the `kubeconfig`
+on your local machine and allow you to interact with your EKS Cluster using `kubectl`.
+
+    ```sh
+    aws eks --region <REGION> update-kubeconfig --name <CLUSTER_NAME>
```
Comment (candonov) [inline docs/getting-started.md:38]:
```suggestion
    aws eks --region <REGION> update-kubeconfig --name <CLUSTER_NAME> --alias <CLUSTER_NAME>
```

File: bootstrap/terraform/variables.tf (hunk 1 above)
Comment (edgarsilva948) [reply bootstrap/terraform/variables.tf:19]: It seems there were changes to CoreDNS in version 1.30 that have affected the ArgoCD deployment. The HAProxy isn't functioning as expected. I'm currently investigating the cause, but in the meantime, I think it's better to revert to the previous version. I'm also setting up end-to-end tests in a separate branch, which should make it easier for us to identify and resolve issues like that.

File: docs/getting-started.md (hunk 2 above)
Comment (edgarsilva948) [reply docs/getting-started.md]: fixed

File: docs/getting-started.md (hunk 3 above)
Comment (edgarsilva948) [reply docs/getting-started.md:27]: fixed

File: docs/getting-started.md (hunk 4 above)
Comment (edgarsilva948) [reply docs/getting-started.md]: fixed

**Review by candonov — APPROVED** [review]
lgtm

**Pull request merged**